Integrations/integration-Remedy_AR.yml

```diff
@@ -0,0 +1,150 @@
+description: BMC Remedy AR System is a professional development environment that leverages
+  the recommendations of the IT Infrastructure Library (ITIL) and provides a foundation
+  for Business Service Management (BSM) solutions.
+configuration:
+- display: Server URL (e.g. https://192.168.0.1)
+  name: server
+  defaultvalue: ""
+  type: 0
+  required: true
+- display: Credentials
+  name: credentials
+  defaultvalue: ""
+  type: 9
+  required: true
+- display: Do not validate server certificate (insecure)
+  name: insecure
+  defaultvalue: "false"
+  type: 8
+  required: false
+script:
+  script: |-
+    ''' IMPORTS '''
+    import requests
+    import json
+    import urllib
+    # disable insecure warnings
+    requests.packages.urllib3.disable_warnings()
+
+    ''' GLOBAL VARS '''
+    SERVER_URL=demisto.params()['server']
+    USERNAME=demisto.params()['credentials']['identifier']
+    PASSWORD=demisto.params()['credentials']['password']
+    BASE_URL=SERVER_URL + '/api'
+    USE_SSL = not demisto.params().get('insecure', False)
+    DEFAULT_HEADERS = {
+        'Content-Type': 'application/json'
+    }
+
+    ''' HELPER FUNCTIONS '''
+    def http_request(method, url_suffix, data, headers):
+        data = {} if data is None else data
+        LOG('running request with url=%s\tdata=%s\theaders=%s' % (BASE_URL + url_suffix,
+            data, headers))
+        try:
+            res = requests.request(method,
+                BASE_URL + url_suffix,
+                verify=USE_SSL,
+                data=data,
+                headers=headers
+            )
+            if res.status_code not in (200, 204):
+                raise Exception('Your request failed with the following error: ' + res.reason)
+        except Exception, e:
+            LOG(e)
```

Comment:
No need to `LOG(e)` here, because the exception will be printed in the `try catch` at the bottom